src/Core/Search/Filters/AttributeGroupFilters.php

```diff
@@ -46,7 +48,7 @@ public static function getDefaults()
             'limit' => 50,
             'offset' => 0,
             'orderBy' => 'position',
```

Comment:
Regarding the "bug" mentioned that the rearrange mode is enabled by default on first access, it is simply because the default column used here is position It should be `id`

Reply:
Unless it's ordered by position on purpose to have a preview of the order in FO, but that's up to @PrestaShop/product-council to tell us that

Reply:
@jolelievre The default order should be just ID, like everywhere else.

Reply:
@Hlavtox @jolelievre, ok done (default order is setting by ID asc)

Reply:
@Hlavtox not all grid handle repositioning, so in regular grids without position it makes sense that the default order is by ID However in entities that can be positioned I think it's better to use the position by default, it reflects right away the state and order of things in the FO
Besides even in legacy page the attributes were already order by position, so I don't understand why you're so fixated on this topic

Reply:
@mattgoud you shouldn't remove this button in the Grid definitions, it will remove both the Done and Rearange button, we only want to remove the Done button

It seems like this is handled by JS here https://github.com/PrestaShop/PrestaShop/blob/4125e4795ff6acfe01bd8c38460941ab6eb349bc/admin-dev/themes/new-theme/js/components/grid/extension/position-extension.ts#L294 For now the extension switched the label of the button, but the idea would be to hide the button when `this.isPositionsReorderActive()` is true

Reply:
> @mattgoud you shouldn't remove this button in the Grid definitions, it will remove both the Done and Rearange button, we only want to remove the Done button
> 
> It seems like this is handled by JS here
> 
> https://github.com/PrestaShop/PrestaShop/blob/4125e4795ff6acfe01bd8c38460941ab6eb349bc/admin-dev/themes/new-theme/js/components/grid/extension/position-extension.ts#L294
> 
> For now the extension switched the label of the button, but the idea would be to hide the button when `this.isPositionsReorderActive()` is true

ok @jolelievre done